mautic/mautic
# Not setting storage_factory_id is deprecated since symfony 5.3.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ x ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ x ]



#### Description:
Replace a deprecated `storage_id` with `storage_factory_id`.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check tests are okay.
3. Check couple of pages that require authorization.
